Cargo.lock

```diff
@@ -707,18 +704,24 @@ dependencies = [
  "uint",
 ]
 
+[[package]]
+name = "halo2_legacy_pdqsort"
+version = "0.1.0"
+source = "registry+https://github.com/rust-lang/crates.io-index"
+checksum = "47716fe1ae67969c5e0b2ef826f32db8c3be72be325e1aa3c1951d06b5575ec5"
+
 [[package]]
 name = "halo2_proofs"
 version = "0.2.0"
```

Comment:
Why not 0.3.0?

Reply:
Oh, because that's blocked on releasing orchard 0.4.0.

Reply:
Actually, you're correct, we can now update these; at the time I initially made these changes the PR updating `orchard` had not yet been merged.

Reply:
Fixed. Depends upon https://github.com/zcash/librustzcash/pull/803